core/src/test/java/io/jenkins/pluginhealth/scoring/probes/DependabotPullRequestProbeTest.java

```diff
@@ -93,19 +98,23 @@ void shouldAccessGitHubAPIAndSeeNoPullRequest() throws IOException {
         final GHLabel dependenciesLabel = mock(GHLabel.class);
         when(dependenciesLabel.getName()).thenReturn("dependencies");
 
+        final Date thirtyDaysAgo = getThirtyDaysAgoAsDate();
+
         final GHPullRequest pr_1 = mock(GHPullRequest.class);
+        doReturn(thirtyDaysAgo).when(pr_1).getCreatedAt();
```

Comment:
as the pull request doesn't have the label `dependencies`, it would have been filtered out first. We don't need to mock the `getCreatedAt` call. 

Or we can add the label `dependencies` on the pull request so that it's not removed from date consideration because it has the label.